src/pages/product-card.js

```
@@ -0,0 +1,25 @@
+import React from 'react';
+
+function ProductCard(props) {
+  const { title, image, price } = props;
+
+  const defaultImage = 'https://source.unsplash.com/phIFdC6lA4E';
```

instead of having a default image can we add logic to hide the card completely if not isn't passed. If there is a name we should show the name. If there is a price we should show a price and so on